src/main/java/uk/nhs/tis/trainee/notifications/service/ProgrammeMembershipService.java

```diff
@@ -55,8 +55,17 @@
 @Service
 public class ProgrammeMembershipService {
 
+  public static final String TIS_ID_FIELD = "tisId";
+  public static final String PERSON_ID_FIELD = "personId";
+  public static final String PROGRAMME_NAME_FIELD = "programmeName";
+  public static final String START_DATE_FIELD = "startDate";
+  public static final String NOTIFICATION_TYPE_FIELD = "notificationType";
+
   private static final String TRIGGER_ID_PREFIX = "trigger-";
-  private static final Integer PAST_MILESTONE_SCHEDULE_DELAY_HOURS = 1;
+  private static final Long PAST_MILESTONE_SCHEDULE_DELAY_HOURS = 1L;
+
+  protected static final Integer WEEKS_AFTER_PROGRAMME_START_MUTE_NOTIFICATIONS = 13;
+  protected static LocalDate notificationServiceLaunchDate = LocalDate.of(2023, 11, 30);
```

Comment:
Another proposed rule - this is quite brittle, but I'm not sure if there's a better option.